src/add-ons/kernel/drivers/nvmm/nvmm.c

```diff
@@ -51,7 +53,7 @@ const struct nvmm_impl *nvmm_impl __read_mostly = NULL;
 struct nvmm_owner nvmm_root_owner;
 
 /* -------------------------------------------------------------------------- */
-
+#if 0
```

Comment:
Why if-0 all this out?

Reply:
I'm if-zeroing everything that I don't need yet. Right now I'm trying to get the initialization of the driver: init_driver() → nvmm_init() → vmx_init() (or svm_init()) working and I haven't needed it yet.